.gitignore

```diff
@@ -87,6 +87,12 @@ celerybeat.pid
 celerybeat-schedule
 
 # Deploy
-conf/Dockerrun.aws.json
+*Dockerrun.aws.json
```

Comment:
This will ignore `Dockerrun.aws.json` no matter what directory it's in - note the * beforehand.